packages/config/src/Config.types.ts

```diff
@@ -78,21 +78,33 @@ export type ExpoGoConfig = {
   };
 };
 
-export type ExpoAppManifest = ExpoConfig &
+export type EASConfig = {
+  projectId?: string;
+};
+
+export type ClientScopingConfig = {
+  scopeKey?: string;
+};
+
+export type ExpoClientConfig = ExpoConfig & {
+  id?: string;
+  releaseId?: string;
+  revisionId?: string;
+  bundleUrl?: string;
+  hostUri?: string;
+  publishedTime?: string;
+};
+
+export type ExpoAppManifest = ExpoClientConfig &
+  EASConfig &
```

Comment:
I could be mistaken but do we ever want to include the EAS config in classic manifests? I suppose it doesn't hurt but wonder if it's needed.